modules/administration/pages/ssl-certs.adoc

```diff
@@ -294,3 +310,47 @@ To complete the process, you need to update the database with this command:
 ----
 
 If you use bootstrap, remember to also update your bootstrap scripts to reflect the new certificate information.
+
+
+
+== Replace Certificates with a Third Party Certificiate
+
+You can replace active certificates on your {productname} installation with a new third party certificate.
+To replace the certificates, you can replace the installed CA certificate RPM with a new RPM containing the third party certificate, and then update the database.
+
+This procedure is similar to the one described in "Create and Replace CA and Server Certificates" with the difference, that we import the certificates generated by an external PKI.
```

Comment:
```suggestion
This procedure is similar to the one described in xref:administration:ssl-certs.adoc#custom-ssl-create-replace-cert[] with the difference, that we import the certificates generated by an external PKI.
```
I'm not sure whether within files we can use the simple `<<custom-ssl-create-replace-cert>>` technique.